<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowTitle">
   <string>–ù–∞—Å—Ç—Ä–æ–π–∫–∏ OCRtoODT</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <widget class="QTabWidget" name="tabWidget">

     <!-- –í–∫–ª–∞–¥–∫–∞ 1: –û–±—â–∏–µ -->
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>–û–±—â–∏–µ</string>
      </attribute>
      <layout class="QFormLayout" name="formLayoutGeneral">
       <item row="0" column="0">
        <widget class="QLabel" name="lblInputDir">
         <property name="text"><string>–ö–∞—Ç–∞–ª–æ–≥ –≤—Ö–æ–¥–Ω—ã—Ö —Ñ–∞–π–ª–æ–≤:</string></property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editInputDir"/>
       </item>

       <item row="1" column="0">
        <widget class="QLabel" name="lblParallel">
         <property name="text"><string>–ü–∞—Ä–∞–ª–ª–µ–ª—å–Ω–∞—è –æ–±—Ä–∞–±–æ—Ç–∫–∞:</string></property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkParallel">
         <property name="text"><string>–í–∫–ª—é—á–∏—Ç—å</string></property>
        </widget>
       </item>

       <item row="2" column="0">
        <widget class="QLabel" name="lblProcesses">
         <property name="text">
          <string>–ö–æ–ª–∏—á–µ—Å—Ç–≤–æ –ø—Ä–æ—Ü–µ—Å—Å–æ–≤:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboProcesses">
         <property name="toolTip">
          <string>–í—ã–±–µ—Ä–∏—Ç–µ –∫–æ–ª–∏—á–µ—Å—Ç–≤–æ –ø–∞—Ä–∞–ª–ª–µ–ª—å–Ω—ã—Ö –ø—Ä–æ—Ü–µ—Å—Å–æ–≤.
–ò—Å–ø–æ–ª—å–∑—É–π—Ç–µ 'auto', —á—Ç–æ–±—ã –∞–≤—Ç–æ–º–∞—Ç–∏—á–µ—Å–∫–∏ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞—Ç—å –≤—Å–µ –¥–æ—Å—Ç—É–ø–Ω—ã–µ —è–¥—Ä–∞ CPU (—Ä–µ–∫–æ–º–µ–Ω–¥—É–µ—Ç—Å—è).</string>
         </property>
         <item><property name="text"><string>auto</string></property></item>
         <item><property name="text"><string>1</string></property></item>
         <item><property name="text"><string>2</string></property></item>
         <item><property name="text"><string>3</string></property></item>
         <item><property name="text"><string>4</string></property></item>
         <item><property name="text"><string>5</string></property></item>
         <item><property name="text"><string>6</string></property></item>
         <item><property name="text"><string>7</string></property></item>
         <item><property name="text"><string>8</string></property></item>
         <item><property name="text"><string>9</string></property></item>
         <item><property name="text"><string>10</string></property></item>
         <item><property name="text"><string>11</string></property></item>
         <item><property name="text"><string>12</string></property></item>
         <item><property name="text"><string>13</string></property></item>
         <item><property name="text"><string>14</string></property></item>
         <item><property name="text"><string>15</string></property></item>
         <item><property name="text"><string>16</string></property></item>
         <item><property name="text"><string>17</string></property></item>
         <item><property name="text"><string>18</string></property></item>
         <item><property name="text"><string>19</string></property></item>
         <item><property name="text"><string>20</string></property></item>
         <item><property name="text"><string>21</string></property></item>
         <item><property name="text"><string>22</string></property></item>
         <item><property name="text"><string>23</string></property></item>
         <item><property name="text"><string>24</string></property></item>
         <item><property name="text"><string>25</string></property></item>
         <item><property name="text"><string>26</string></property></item>
         <item><property name="text"><string>27</string></property></item>
         <item><property name="text"><string>28</string></property></item>
         <item><property name="text"><string>29</string></property></item>
         <item><property name="text"><string>30</string></property></item>
         <item><property name="text"><string>31</string></property></item>
         <item><property name="text"><string>32</string></property></item>
        </widget>
       </item>
      </layout>
     </widget>

     <!-- –í–∫–ª–∞–¥–∫–∞ 2: –ü—Ä–µ–¥–æ–±—Ä–∞–±–æ—Ç–∫–∞ -->
     <widget class="QWidget" name="tabPreprocess">
      <attribute name="title"><string>–ü—Ä–µ–¥–æ–±—Ä–∞–±–æ—Ç–∫–∞</string></attribute>
      <layout class="QVBoxLayout" name="layoutPreproc">
       <item><widget class="QCheckBox" name="chkGrayscale"><property name="text"><string>–ü—Ä–µ–æ–±—Ä–∞–∑–æ–≤–∞—Ç—å –≤ –æ—Ç—Ç–µ–Ω–∫–∏ —Å–µ—Ä–æ–≥–æ</string></property></widget></item>
       <item><widget class="QCheckBox" name="chkDenoise"><property name="text"><string>–£–¥–∞–ª–∏—Ç—å —à—É–º (–º–µ–¥–∏–∞–Ω–Ω—ã–π —Ñ–∏–ª—å—Ç—Ä)</string></property></widget></item>
       <item><widget class="QCheckBox" name="chkContrast"><property name="text"><string>–ü–æ–≤—ã—Å–∏—Ç—å –∫–æ–Ω—Ç—Ä–∞—Å—Ç (CLAHE)</string></property></widget></item>
       <item><widget class="QCheckBox" name="chkBinarize"><property name="text"><string>–ë–∏–Ω–∞—Ä–∏–∑–∞—Ü–∏—è (Otsu)</string></property></widget></item>
       <item><widget class="QCheckBox" name="chkSharpen"><property name="text"><string>–†–µ–∑–∫–æ—Å—Ç—å –∫—Ä–∞—ë–≤</string></property></widget></item>
      </layout>
     </widget>

     <!-- –í–∫–ª–∞–¥–∫–∞ 3: –†–∞—Å–ø–æ–∑–Ω–∞–≤–∞–Ω–∏–µ -->
     <widget class="QWidget" name="tabRecognition">
      <attribute name="title"><string>–†–∞—Å–ø–æ–∑–Ω–∞–≤–∞–Ω–∏–µ</string></attribute>
      <layout class="QFormLayout" name="formLayoutRecog">
       <item row="0" column="0">
        <widget class="QLabel" name="lblLanguage"><property name="text"><string>–Ø–∑—ã–∫ Tesseract:</string></property></widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboLang">
         <item><property name="text"><string>rus+eng</string></property></item>
         <item><property name="text"><string>rus</string></property></item>
         <item><property name="text"><string>eng</string></property></item>
        </widget>
       </item>

       <item row="1" column="0">
        <widget class="QLabel" name="lblIndentMin"><property name="text"><string>–ú–∏–Ω. –æ—Ç—Å—Ç—É–ø –∞–±–∑–∞—Ü–∞:</string></property></widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="spinIndentMin">
         <property name="value"><double>2.0</double></property>
         <property name="singleStep"><double>0.5</double></property>
         <property name="suffix"><string> –º–º</string></property>
        </widget>
       </item>

       <item row="2" column="0">
        <widget class="QLabel" name="lblIndentMax"><property name="text"><string>–ú–∞–∫—Å. –æ—Ç—Å—Ç—É–ø –∞–±–∑–∞—Ü–∞:</string></property></widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="spinIndentMax">
         <property name="value"><double>10.0</double></property>
         <property name="singleStep"><double>0.5</double></property>
         <property name="suffix"><string> –º–º</string></property>
        </widget>
       </item>

        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="chkNotifyOnFinish">
          <property name="text">
           <string>–ü–æ–∫–∞–∑—ã–≤–∞—Ç—å –æ–∫–Ω–æ —É–≤–µ–¥–æ–º–ª–µ–Ω–∏—è –ø–æ—Å–ª–µ –∑–∞–≤–µ—Ä—à–µ–Ω–∏—è OCR</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="chkSoundOnFinish">
          <property name="text">
           <string>–ü—Ä–æ–∏–≥—Ä—ã–≤–∞—Ç—å –∑–≤—É–∫ –ø–æ—Å–ª–µ –∑–∞–≤–µ—Ä—à–µ–Ω–∏—è OCR</string>
          </property>
         </widget>
        </item>

        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="btnTestSound">
          <property name="text">
           <string>üîä –¢–µ—Å—Ç –∑–≤—É–∫–∞</string>
          </property>
         </widget>
        </item>



      </layout>
     </widget>

     <!-- –í–∫–ª–∞–¥–∫–∞ 4: ODT -->
     <widget class="QWidget" name="tabODT">
      <attribute name="title"><string>ODT</string></attribute>
      <layout class="QFormLayout" name="formLayoutODT">
       <item row="0" column="0"><widget class="QLabel" name="lblFont"><property name="text"><string>–®—Ä–∏—Ñ—Ç:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboFont">
         <item><property name="text"><string>Times New Roman</string></property></item>
         <item><property name="text"><string>Arial</string></property></item>
         <item><property name="text"><string>Calibri</string></property></item>
        </widget>
       </item>

       <item row="1" column="0"><widget class="QLabel" name="lblMargins"><property name="text"><string>–ü–æ–ª—è (–º–º):</string></property></widget></item>
       <item row="1" column="1"><widget class="QLineEdit" name="editMargins"><property name="placeholderText"><string>20, 15, 20, 15</string></property></widget></item>

       <item row="2" column="0"><widget class="QLabel" name="lblAlign"><property name="text"><string>–í—ã—Ä–∞–≤–Ω–∏–≤–∞–Ω–∏–µ:</string></property></widget></item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboAlign">
         <item><property name="text"><string>–ü–æ –ª–µ–≤–æ–º—É –∫—Ä–∞—é</string></property></item>
         <item><property name="text"><string>–ü–æ —Ü–µ–Ω—Ç—Ä—É</string></property></item>
         <item><property name="text"><string>–ü–æ —à–∏—Ä–∏–Ω–µ</string></property></item>
        </widget>
       </item>

       <item row="3" column="0"><widget class="QLabel" name="lblDefGap"><property name="text"><string>–ü–æ—Ä–æ–≥ —Ä–∞–∑—Ä—ã–≤–∞ –∞–±–∑–∞—Ü–∞:</string></property></widget></item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="spinDefGap">
         <property name="value"><double>1.20</double></property>
         <property name="singleStep"><double>0.1</double></property>
         <property name="suffix"><string> –º–º</string></property>
        </widget>
       </item>
      </layout>
     </widget>

    <!-- –í–∫–ª–∞–¥–∫–∞ 5: –ò–Ω—Ç–µ—Ä—Ñ–µ–π—Å -->
    <widget class="QWidget" name="tabInterface">
     <attribute name="title"><string>–ò–Ω—Ç–µ—Ä—Ñ–µ–π—Å</string></attribute>
     <layout class="QFormLayout" name="formLayoutUI">

      <item row="0" column="0">
       <widget class="QLabel" name="lblTheme">
        <property name="text"><string>–¢–µ–º–∞ –æ—Ñ–æ—Ä–º–ª–µ–Ω–∏—è:</string></property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboTheme">
        <item><property name="text"><string>–°–≤–µ—Ç–ª–∞—è</string></property></item>
        <item><property name="text"><string>–¢—ë–º–Ω–∞—è</string></property></item>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="lblFontSize">
        <property name="text"><string>–†–∞–∑–º–µ—Ä —à—Ä–∏—Ñ—Ç–∞ –ª–æ–≥–æ–≤:</string></property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinFontSize">
        <property name="minimum"><number>8</number></property>
        <property name="maximum"><number>20</number></property>
        <property name="value"><number>10</number></property>
       </widget>
      </item>

     </layout>
    </widget>

    </widget>
   </item>

   <!-- –ù–∏–∂–Ω–∏–µ –∫–Ω–æ–ø–∫–∏ -->
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item><spacer name="horizontalSpacer"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
     <item><widget class="QPushButton" name="btnOK"><property name="text"><string>OK</string></property></widget></item>
     <item><widget class="QPushButton" name="btnCancel"><property name="text"><string>–û—Ç–º–µ–Ω–∞</string></property></widget></item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
